bitnami/redis/templates/redis-slave-statefulset.yaml

```diff
@@ -481,7 +481,7 @@ spec:
     {{- if not .Values.slave.persistence.enabled }}
         - name: redis-data
           emptyDir: {}
-    {{- else }}
+    {{- end }}
```

Comment:
I think the `else` here was doing fine. Instead, by moving the volume above the `if-else` statement, the problem would be solved 😁. 
```diff
+   {{- if .Values.tls.enabled }}
+       - name: redis-certificates
+         secret:
+           secretName: {{ required "A secret containing the certificates for the TLS traffic is required when TLS in enabled" .Values.tls.certificatesSecret }}
+           defaultMode: 256
+   {{- end }}
    {{- if not .Values.slave.persistence.enabled }}
        - name: redis-data
          emptyDir: {}
    {{- else }}
-    {{- if .Values.tls.enabled }}
-        - name: redis-certificates
-          secret:
-            secretName: {{ required "A secret containing the certificates for the TLS traffic is required when TLS in enabled" .Values.tls.certificatesSecret }}
-            defaultMode: 256
-    {{- end }}
  volumeClaimTemplates:
```

Could you try to do that?

Reply:
Thanks for the suggestion. :) I'm not used to YAML conditional and I got a bit confused by the indentation. Let me try that.